lib/src/components/NxFormSelect/__tests__/NxFormSelect.test.tsx

```diff
@@ -5,45 +5,490 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
+import userEvent from '@testing-library/user-event';
 
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxFormSelect, { Props } from '../NxFormSelect';
-import { mount } from 'enzyme';
+import { render, within } from '@testing-library/react';
+import NxForm from '../../NxForm/NxForm';
 
 describe('NxFormSelect', function() {
   const minimalProps = { value: '', onChange: () => {} },
-      getShallow = getShallowComponent<Props>(NxFormSelect, minimalProps);
+      quickRender = rtlRender(NxFormSelect, minimalProps),
+      renderEl = rtlRenderElement(NxFormSelect, minimalProps);
 
-  it('renders a <select> with the nx-form-select class', function() {
-    expect(getShallow()).toMatchSelector('select.nx-form-select');
+  it('renders a <select> ', function() {
+    expect(quickRender().getByRole('combobox').tagName).toBe('SELECT');
```

Comment:
do selects have an inherit role of combobox, or is it specified somewhere in NxFormSelect?

Reply:
> do selects have an inherit role of combobox, or is it specified somewhere in NxFormSelect?

It's built in to HTML.  See https://www.w3.org/TR/html-aria/#docconformance